src/actions/account.js

```diff
@@ -3,26 +3,17 @@ import { createActions } from 'redux-actions'
 import { Wallet } from '../utils/wallet'
 
 export const REFRESH_ACCOUNT = 'REFRESH_ACCOUNT'
-export const LOADER_ACCOUNT = 'LOADER_ACCOUNT'
 export const REFRESH_URL = 'REFRESH_URL'
 
 export function handleRefreshAccount(history, loader = true) {
    return (dispatch, getState) => {
-      if (loader) {
-         dispatch({
-            type: LOADER_ACCOUNT,
-            loader: true
-         })   
-      }
+      dispatch(loginPending(loader))
```

Comment:
Any reason to not go with async function (similar to wallet actions)? Seems like it'll allow to make this method simpler as request lifecycle management will be handled by middleware and you'll only need to dispatch another action for clearing accounts.